deb-package/pkg/etc/dedis/dvoting/config.env

```diff
@@ -0,0 +1,12 @@
+# dela config
+export dela_bin="/opt/dedis/dvoting/bin/memcoin"
+export dela_data="/var/opt/dedis/dvoting/data/dela"
+export dela_port="9000"
+export prometheus_addr="127.0.0.1:9100"
+export proxy_addr="127.0.0.1:9080"
```

Comment:
I believe these 2 lines ensure that prometheus and the proxy will *not* be accessible remotely, as they're binding to the loopback address. We should probably bind to `0.0.0.0:9100` and `0.0.0.0:9080` respectively. Am I mistaken?